src/index.js

```diff
@@ -1,4 +1,5 @@
-import React, { StrictMode } from 'react';
+import { StrictMode } from 'react';
+import * as React from 'react';
```

Comment:
Maybe this can be solved in one line, maybe it was fine as it was?

Reply:
probably as it was